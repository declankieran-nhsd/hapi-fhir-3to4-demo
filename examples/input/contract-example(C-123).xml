<?xml version="1.0" encoding="UTF-8"?><Contract xmlns="http://hl7.org/fhir">

   <text>
      <status value="generated"/>
      <div xmlns="http://www.w3.org/1999/xhtml">A human-readable rendering of the contract</div>
   </text>
   
   <identifier>
      <system value="http://happyvalley.com/contract"/>
      <value value="12347"/>
   </identifier>
   
   <!-- 
  <issued value="2014-08-17"/>
 
  <subject>
     <reference value="http://www.somewhere.org/Patient/12345"/>
  </subject>

  <type>
     <coding>
	   <system value="http://hl7.org/fhir/contracttypecodes"/>
	   <code value="disclosure"/>
	 </coding>
  </type>

  <subtype>
     <coding>
	   <system value="http://hl7.org/fhir/contractsubtypecodes"/>
	   <code value="disclosure-US"/>
	 </coding>
  </subtype>
  -->
</Contract>