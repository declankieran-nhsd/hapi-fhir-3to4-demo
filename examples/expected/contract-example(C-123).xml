<?xml version="1.0" encoding="UTF-8"?>

<Contract xmlns="http://hl7.org/fhir">
  <text>
    <status value="generated"/><div xmlns="http://www.w3.org/1999/xhtml">A human-readable rendering of the contract</div>
  </text>
  <identifier>
    <system value="http://happyvalley.com/contract"/>
    <value value="12347"/>
  </identifier>
</Contract>